05_iigo/index.tex

```diff
@@ -948,7 +948,64 @@ \subsection{Rule Representation}
 \end{rule_IIGO}
 We then conditionally execute them such that if Rule~\ref{rule:cond1} fails then the whole structure passes, and if Rule~\ref{rule:cond1} passes, we expect Rule~\ref{rule:cond2} to pass as well for the rule to be adhered.
 
-\section{Future Work}
+\section{Analysis and Reflection}
+\subsection{Ostrom’s Design Principles and IIGO}
+\subsubsection{Institutional Analysis and Development framework overview of IIGO}
+
+In this section, we start the evaluation of IIGO by briefly stating or re-visiting the institution's \emph{Bio-Physical Characteristics}, \emph{Attributes of the Community}, \emph{Rules-in-Use}, as well as defining the distinct \emph{Action Arenas} and relevant \emph{Actors}.
+
+\begin{itemize}
+    \item Bio-Physical Characteristics
+    
+    Resources are stored in two distinct \emph{facilities}: in private pools and in the common pool. The only \emph{facility} that can generate resources is foraging. Resources are expended through the cost of living, disaster events, cost of IIGO actions and potentially foraging. In the scope of this coursework, IIGO is only concerned with managing the movement of resources between private pools and the common pool. 
```

Comment:
I would bold the item titles here and put a colon or the formatting can be a bit unpredictable